src/test/java/Lesson10.java

```diff
@@ -0,0 +1,110 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class Lesson10 {
+    WebDriver driver;
+    private static final String HEROCU_APP_URL_CONTEXT_MENU = "http://the-internet.herokuapp.com/context_menu";
```

Comment:
Try to avoid code duplication. Use: 
HEROKU_APP_URL = http://the-internet.herokuapp.com
CONTEXT_MENU_URL = HEROKU_APP_URL + "/context_menu"

and so on for other constants

Reply:
Done